src/css/footer.css

```diff
@@ -10,6 +10,103 @@ footer.footer {
   margin: 0.5rem 0;
 }
 
-.footer a {
-  color: var(--footer-link-font-color);
+.footer-info {
+  display: flex;
+  gap: 24px;
+}
+
+@media screen and (max-width: 720px) {
+  .footer-info {
+    flex-direction: column;
+  }
+}
+
+@media screen and (min-width: 720px) {
+  .footer-info {
+    flex-direction: row;
+  }
+}
+
+.footer-info-item-large {
+  flex-grow: 3;
+}
+
+.footer-info-item {
+  flex-grow: 1;
+}
+
+.footer-info-item h2 {
+  margin-top: 0;
+}
+
+.footer-link-list {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.footer-link-list li a {
+  color: var(--footer-font-color);
+}
+
+.footer-link-list li a:hover {
+  text-decoration: none;
+  color: var(--color-brand-primary);
+}
+
+.c-and-socials {
+  display: flex;
+  gap: 24px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+@media screen and (max-width: 720px) {
+  .c-and-socials {
+    flex-direction: column;
+  }
+}
+
+@media screen and (min-width: 720px) {
+  .c-and-socials {
+    flex-direction: row;
+  }
+}
+
+.footer-container {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.social-icons-container {
+  display: flex;
+  flex-direction: row;
+  gap: 10px;
+}
+
+.social-icon {
+  background-color: var(--footer-font-color);
+  color: var(--body-font-color);
+  border-radius: 50%;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
**Optional:** This should not be necessary.

Reply:
Just to make sure I got it right, you are referring to `justify-content: center`, correct? I removed it and the icons look like this: 
![image](https://user-images.githubusercontent.com/5787489/213429252-b7305c5c-c75d-4714-84b6-f0465cd5f710.png)